source/tests/amc_uploader/test_amc_uploader.py

```diff
@@ -9,13 +9,17 @@ def mock_env_variables():
     os.environ["AMC_ENDPOINT_URL"] = "AmcEndpointUrl"
     os.environ["AMC_API_ROLE_ARN"] = "AmcApiRoleArn"
     os.environ["botoConfig"] = '{"region_name": "us-east-1"}'
+    os.environ["SOLUTION_VERSION"] = '0.01'
```

Comment:
Version should look like a semantic version (number dot number dot number). I typically use "0.0.0" for the version on test stacks.

Reply:
i will just use the one u use "0.0.0"